components/Card.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <NuxtLink :to="`/products/${items.id}`">
+    <UCard class="w-[100%] max-h-[400px] overflow-hidden" @click="handleCardClick(items.id)">
```

Comment:
You could use `w-full` instead of `w-[100%]` 
You could also remove `handleCardClick` as you are routing through NuxtLink

Reply:
@archit-credilio I have resolved the comment

Reply:
We can use `w-full` as it is a direct tailwind class. We use `w-[arbitraryValue]` syntax only when there's no predefined class for it. example
w-[332px]

Reply:
resolved